Source/WebKit/UIProcess/mac/WebViewImpl.mm

```diff
@@ -2408,6 +2407,7 @@ static NSTrackingAreaOptions trackingAreaOptions()
 {
     m_shouldDeferViewInWindowChanges = true;
     viewWillMoveToWindowImpl(targetWindow);
+    [m_windowVisibilityObserver setIsPendingDuplicateObserverRemoval:YES];
```

Comment:
```suggestion
    m_windowVisibilityObserver.pendingDuplicateObserverRemoval = YES;
```

Reply:
(This one won't quite work, since `m_windowVisibilityObserver` is a RetainPtr; our preference in WebKit is generally to use `setIsPendingDuplicateObserverRemoval:` instead of `.get().pendingDuplicateObserverRemoval`)